app/services/cycle_timetable_query.rb

```diff
@@ -95,7 +74,7 @@ def self.next_cycle_year
 
   def self.schedules
     {
-      real: CYCLE_DATES[CURRENT_YEAR_FOR_SCHEDULE],
+      real: RealCycleSchedule.new(RECRUITMENT_CYCLE_YEAR).cycle_dates,
```

Comment:
Should we extract the other schedules to classes also so that we have the 'same level of abstraction'?

Reply:
Not sure about this change — pulling what's currently a single list of dates out into separate files isn't necessarily an improvement to readability/debuggability?

Reply:
Yeah that's a good point. I guess the rationale for extracting the other schedules is more about consistency and 
not causing any 'surprises' ... but on the whole may not be worth it.

The reason for extracting the 'CYCLE_DATES' hash into a single class (the poorly named `RealCycleSchedule` class) was because of the duplication (and I think forms the main rationale for this ticket 🤔 ). However if the consensus is that we rather stick to the 'CYCLE_DATES' hash (and add a sub hash for each subsequent RecruitmentCycleYear) then possibly this ticket is defunct?